docs/source/teams/api_connection.rst

```diff
@@ -30,9 +30,9 @@ FiftyOne config as described below:
 +-------------------------------+-------------------------------------+----------------------------------------------------------------------------------------+
 | Config field                  | Environment variable                | Description                                                                            |
 +===============================+=====================================+========================================================================================+
-| `api_uri`                     | `FIFTYONE_API_URI`                  | The URI of your FiftyOne Teams API. Ask your deployment admin for this value.          |
+| `api_uri`                     | `FIFTYONE_API_URI`                  | The URI of your FiftyOne Enterprise API. Ask your deployment admin for this value.          |
 +-------------------------------+-------------------------------------+----------------------------------------------------------------------------------------+
-| `api_key`                     | `FIFTYONE_API_KEY`                  | Your FiftyOne Teams API key. :ref:`See here <teams-generate-api-key>` to generate one. |
+| `api_key`                     | `FIFTYONE_API_KEY`                  | Your FiftyOne Enterprise API key. :ref:`See here <teams-generate-api-key>` to generate one. |
```

Comment:
These tables are malformed 😢

Reply:
fixed

Reply:
augh thank you. At least the docs built this time :D progress!